std/flags/mod.ts

```diff
@@ -108,7 +108,7 @@ export function parse(
     stopEarly = false,
     string = [],
     unknown = (i: unknown): unknown => i
-  }: Partial<ArgParsingOptions> = {}
```

Comment:
Isn't `Partial<T>` equivalent of this change?

Reply:
I put it at the interface definition for consistency and because it's a more correct specification of the options. If you had to build up the options object slowly before calling parse, you might initialise it like `const argParsingOptions = {};` and not populate every option.